operator/params.yaml

```diff
@@ -60,6 +60,14 @@ parameters:
     description: "When a pod starts and the probe fails, `failure_threshold` attempts will be made before marking the pod as 'unready'."
     default: "3"
 
+  - name: NODE_DATACENTER
```

Comment:
We might not want the user to set these parameters. I'm currently figuring out the [cluster's topology](https://kubernetes.io/docs/reference/kubernetes-api/labels-annotations-taints/#topologykubernetesiozone) should and can be used instead.

Reply:
Yeah, this is a bit too simple. We want to support setups where, e.g., 2 nodes are in rack 1 and another node is in rack 2. This demands a more sophisticated configuration and would result in multiple `StatefulSets`, one for each datacenter/rack.